cirq/google/calibration/phased_fsim.py

```diff
@@ -220,6 +220,22 @@ class PhasedFSimCalibrationRequest(abc.ABC):
     pairs: Tuple[Tuple[Qid, Qid], ...]
     gate: Gate  # Any gate which can be described by cirq.PhasedFSim
 
+    @staticmethod
+    def _create_pairs_from_moment(moment: Moment):
+        gate = None
+        pairs = []
+        for op in moment:
+            if gate is None:
+                gate = op.gate
```

Comment:
op.gate is optional, isn't it possible that the first operation will not have a gate and will sneak through this code?

Reply:
Done.